manifest.json

```
@@ -0,0 +1,20 @@
+{
+  "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "id": "tls-paranoia@gdr.name",
+      "strict_min_version": "68.0"
+    }
+  },
+  "name": "Paranoia",
+  "description": "Check if your emails arrived encrypted (and which corporations were able to read them nevertheless)",
+  "version": "0.2.7",
```

I think it needs a version bump for uploading to https://addons.thunderbird.net/en-US/thunderbird/addon/paranoia/

```suggestion
  "version": "0.3.0",
```

---

done